controller/scheduler-v2/scheduler.go

```diff
@@ -62,63 +69,71 @@ func (s *Scheduler) Run() error {
 	log.Info("starting scheduler loop")
 	defer log.Info("exiting scheduler loop")
 
-	for {
-		// first, check if we should stop or process pending job events
-		select {
-		case <-s.stop:
-			return nil
-		case req := <-s.jobRequests:
-			s.HandleJobRequest(req)
-			continue
-		default:
-		}
-
-		log.Info("starting cluster sync")
-		if err := s.Sync(); err != nil {
-			log.Error("error performing cluster sync", "err", err)
-			continue
+	jobTicker := time.Tick(30 * time.Second)
+	formationTicker := time.Tick(time.Minute)
+	formationTimeout := time.After(time.Second)
+	go func() {
+		for {
+			select {
+			case <-jobTicker:
+				triggerChan(s.jobSync)
+			case <-formationTicker:
+				triggerChan(s.formationSync)
+			case <-formationTimeout:
+				triggerChan(s.formationSync)
+			}
```

Comment:
We should just select on the ticker channels in the main loop and drop this goroutine.